README.md

```diff
@@ -279,3 +279,7 @@ All the CFPs (past and coming) are available publicly in JSON: https://developer
 
 * 17-19: [Devoxx France](https://www.devoxx.fr/) - Paris (France)
 * 25-26: [MiXiT](https://mixitconf.org/fr/) - Villeurbanne (Lyon France)
+
+### March
+
+* 19-22: [KubeCon + CloudNativeCon Europe 2024](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2024/) - Paris (France)
```

Comment:
Thanks!
Just a remark, is it possible to copy paste this new March section before April? 😇
Thx🥰

Reply:
Done! I should learn the months 😅